packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -1110,3 +1116,43 @@ class WrappableHeadersTable extends React.Component {
   }
 }
 ```
+
+## Empty state table
+
+```js
+import React from 'react';
+import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { Button, EmptyState, EmptyStateBody, EmptyStatePrimary, Bullseye } from '@patternfly/react-core';
+
+EmptyStateTable = () => {
+  const columns = ['Repositories', 'Branches', 'Pull request', 'Workspaces', 'LastCommit']
+  const rows = [{
+    heightAuto: true,
```

Comment:
@priley86 what do you think about introducing row decorators? Similiar to how cell formatters work. This can be potentially problem in future when we'll update API and we will have to keep it intact.